website/src/redux/reducers/ElementReducer.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file Reducers having to do with the position of elements.
+ */
+
+import {
+  INIT_ELEMENTS, UPDATE_POSITION, UPDATE_SIZE, UPDATE_ROTATION, TARGET_ELEMENT,
+} from './../actions/ActionConstants';
+
+/**
+ * Inserts payload information into state, making a new object.
+ * @param {Object} state The state of the store.
+ * @param {Object} elementId ID of the element affected.
+ * @param {Object} payload The updated information to insert.
+ * @param {String} objectField The field of the element to update. If left as
+ *                             null this indicates the payload already contains
+ *                             the field.
+ * @returns {Object} The new state object.
+ */
+function copyPayloadInfo(state, elementId, payload, objectField = null) {
```

Comment:
objectField could be named/explained better. It is unclear as to what it does. Ex: update rotation doesn't require this param since it is not a nested obj, but that isn't clear without looking at lines 25-29